### ISSUE TITLE
ImageNet of Validation

### ISSUE THREAD

**Issue by cyy280113999** [naver-ai/relabel_imagenet#12]
Oh no. I thought i could use the relabeled dataset to test my model, so i download imagenet val in 10GB. 
But it is that the relabeled is for the training split which is 100GB. 

Do you have relabeled val? I am doing the downloading.

BTW. Is ImageNet 2012? Does train and val overlap in samples?

**Comment by hellbell** [comment]
Oh, our project only deals with `training data` 😢 .
These repos may help you. They provide re-labeled val data.

- Are we done with ImageNet? [[paper]](https://arxiv.org/abs/2006.07159) [[github]](https://github.com/google-research/reassessed-imagenet)
- Evaluating Machine Accuracy on ImageNet (ICML'20) [[paper]](http://proceedings.mlr.press/v119/shankar20c.html) [[github]](https://github.com/modestyachts/evaluating_machine_accuracy_on_imagenet)

**Comment by emilywenger** [comment]
Would you be open to sharing the model you used to relabel Imagenet, or using this model to label the validation set for sharing? It would be extremely helpful to have the multi-label validation dataset available alongside the training dataset.

**Comment by hellbell** [comment]
@emilywenger 

> the model you used to relabel Imagenet

We used `tf_efficientnet_l2_ns_475` from timm repository (https://github.com/rwightman/pytorch-image-models).

> using this model to label the validation set

Validation is not our purpose. If you need refined validation set for imagenet, see https://github.com/naver-ai/relabel_imagenet/issues/12#issuecomment-1259638476.
